apps/judicial-system/web/src/components/CourtDocuments/CourtDocuments.tsx

```diff
@@ -3,19 +3,17 @@ import { useIntl } from 'react-intl'
 import Select, {
   ClearIndicatorProps,
   components,
-  ControlProps,
   DropdownIndicatorProps,
-  MenuProps,
   OptionProps,
   PlaceholderProps,
-  ValueContainerProps,
+  SingleValueProps,
 } from 'react-select'
-import cn from 'classnames'
 
-import { Box, Icon, Tag, Text } from '@island.is/island-ui/core'
+import { Box, IconDeprecated, Tag, Text } from '@island.is/island-ui/core'
```

Comment:
FYI: I'm using the deprecated icon component because the non-deprecated one keeps flickering on state change.